src/components/DatePicker/RelativeDateRange.jsx

```diff
@@ -220,13 +242,23 @@ export const RelativeDateRange = ({
     onChange,
     onSubmit,
     timezoneName,
+    isFutureDatesAllowed = false, // This prop used to be able to select Future Dates in RelativeDateRange,
 }) => {
     const handleChange = (e) => {
         const rangeName = e.target.value;
         const range = handlers[rangeName](timezoneName);
         onChange(rangeName, range);
     };
 
+    const filterFutureDates = (options) => {
```

Comment:
this func. we are try moving out from component

Reply:
yes please. move it out of the component.